ankerctl.py

```diff
@@ -458,7 +458,7 @@ def webserver(env):
 
 @app.get("/")
 def app_root():
-    return render_template("index.html", configPort = app.config["port"], configHost = app.config["host"])
+    return render_template("index.html", requestPort = request.host.split(':')[1], requestHost = request.host.split(':')[0])
```

Comment:
The flask documentation states (https://tedboy.github.io/flask/generated/generated/flask.Request.host.html#flask.Request.host)

```
Request.host

    Just the host including the port if available. See also: trusted_hosts.
```

So when running on a standard port, this will make the application crash and become non-functional.

Flask uses werkzeug, with the relevant parts in `werkzeug/src/werkzeug/sansio/request.py` and `werkzeug/src/werkzeug/sansio/utils.py`. Maybe we can get inspiration there.

Can you suggest a more robust solution? I like the idea overall :)

Reply:
It doesn't cause the application to crash, just the webpage when sourced at `/` (returns a 500); however, I did find a solution for this and have pushed the necessary change.